gpt_engineer/learning.py

```diff
@@ -228,15 +228,16 @@ def extract_learning(
     learning = Learning(
         prompt=dbs.input["prompt"],
         model=model,
-        temperature=temperature,
+        temperature=round(temperature, 2),
```

Comment:
Why does the temperature need to be rounded here?  It feels irrelevant for fixing this issue.